themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2529,6 +2529,610 @@ Status: 200 OK
     "AWS_SESSION_TOKEN": "<redacted>""
 ```
 
+## Schedules
+
+{{< notes >}}
+Pulumi Schedules REST API endpoints are currently in private preview.
+{{< /notes >}}
+
+<!-- ###################################################################### -->
+
+### List Schedules of a stack
+
+```
+GET /api/stacks/{organization}/{project}/{stack}/deployments/schedules
+```
+
+#### Parameters
+
+| Parameter           | Type   | In    | Description                                                                                                  |
+|---------------------|--------|-------|--------------------------------------------------------------------------------------------------------------|
+| `organization`      | string | path  | organization name                                                                                            |
+| `project`           | string | path  | project name                                                                                                 |
+| `stack`             | string | path  | stack name                                                                                                   |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/deployments/schedules
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "scheduledDeployments": [
+    {
+      "id": "12345678-8102-447f-b246-e9ec85786e23",
+      "orgID": "87654321-8b3b-418a-8c01-5ddd0e00bace",
+      "scheduleCron": "0 0 * * *",
+      "nextExecution": "2024-04-20 00:00:00.000",
+      "paused": false,
+      "kind": "deployment",
+      "definition": {
+        "programID": "12345678-9f94-4a92-8bfb-c2da724fa65b",
+        "request": {
+          "operation": "update"
+        }
+      },
+      "created": "2024-04-19 01:00:00.000",
+      "modified": "2024-04-19 01:00:00.000",
+      "lastExecuted": null
+    }
+  ]
+}
+
+```
+
+### Create Drift Schedule
+
```

Comment:
Would be great to add a note here and Update Drift Schedule and Create/Update TTL Schedule that says something like:

```
This is a convenience API for ease of use to create/update [this] schedule, but the same configuration is possible via the raw deployments schedule APIs. 
```